src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
variable inside class Ball related to this class, u can name it : collor and number, instead of ballCollor and ballNumber